src/lib/ops/policyOps.js

```diff
@@ -145,18 +145,35 @@ const policyOps = {
    * @param  {Function} cb
    */
   listByOrganization: function listByOrganization (params, cb) {
-    const { organizationId } = params
+    const { organizationId, limit, page } = params
 
```

Comment:
they say in doc: `OFFSET 0 is the same as omitting the OFFSET clause, and LIMIT NULL is the same as omitting the LIMIT clause`